packages/node_modules/@webex/plugin-meetings/src/statsAnalyzer/index.js

```diff
@@ -107,6 +120,9 @@ export default class StatsAnalyzer extends EventsScope {
       isShareStream: true
     });
 
+    mqaData.intervals[0].intervalMetadata.peerReflexiveIP = this.statsResults.connectionType.local.ipAddress[0];
```

Comment:
Do we need to check if the `intervals` array exists and has data?

Reply:
There is a global file which has a meta data for the intervals array